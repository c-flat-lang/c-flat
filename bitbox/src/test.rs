// use super::*;
//
// #[test]
// fn test_passes() {}
