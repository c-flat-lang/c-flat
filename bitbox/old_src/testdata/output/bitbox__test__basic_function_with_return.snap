---
source: bitbox/src/test.rs
expression: contents
---
function add(_x_v0_, _y_v0_, ) u32{
    @add u32 : _tmp0_v0_, _x_v0_, _y_v0_
    @ret u32, _tmp0_v0_
}
